src/azul/__init__.py

```diff
@@ -93,6 +93,14 @@ def es_index(self) -> str:
     def domain_name(self) -> str:
         return os.environ['AZUL_DOMAIN_NAME']
 
+    @property
+    def git_commit_sha(self) -> str:
+        return os.environ['GIT_COMMIT_SHA']
+
+    @property
+    def git_repo_dirty(self) -> str:
+        return os.environ['GIT_REPO_DIRTY']
```

Comment:
Convert string back to bool. I want the JSON response to be `"dirty": true`, not `"dirty": "True"`.